simbad/command_line/simbad_full.py

```diff
@@ -61,7 +61,7 @@ def main():
         raise OSError("amore executable not found")
 
     gui = simbad.util.pyrvapi_results.SimbadOutput(
-        args.rvapi_document, args.webserver_uri, args.display_gui, log_file, args.work_dir
+        args.rvapi_document, args.webserver_uri, args.display_gui, log_file, args.work_dir, ccp4i2_xml=args.ccp4i2_xml
```

Comment:
Signature of the constructor is:
```def __init__(self, rvapi_document, webserver_uri, display_gui, logfile, work_dir, ccp4i2_xml):```

Either we make it an optional keyword argument or we remove the keyword argument everywhere. One or the other ...